README.md

```diff
@@ -78,57 +78,35 @@ HOSTNAME_COMMAND=wget -t3 -T2 -qO-  http://169.254.169.254/latest/meta-data/loca
 ```
 Reference: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html
 
-## Broker Rack
-
-You can configure the broker rack affinity in different ways
-
-1. explicitly, using ```KAFKA_BROKER_RACK```
-2. via a command, using ```RACK_COMMAND```, e.g. ```RACK_COMMAND: "curl http://169.254.169.254/latest/meta-data/placement/availability-zone"```
-
-In the above example the AWS metadata service is used to put the instance's availability zone in the ```broker.rack``` property.
-
-## JMX
-
-For monitoring purposes you may wish to configure JMX. Additional to the standard JMX parameters, problems could arise from the underlying RMI protocol used to connect
-
-* java.rmi.server.hostname - interface to bind listening port
-* com.sun.management.jmxremote.rmi.port - The port to service RMI requests
+### Injecting HOSTNAME_COMMAND into configuration
 
-For example, to connect to a kafka running locally (assumes exposing port 1099)
+If you require the value of `HOSTNAME_COMMAND` in any of your other `KAFKA_XXX` variables, use the `_{HOSTNAME_COMMAND}` string in your variable value, i.e.
 
-      KAFKA_JMX_OPTS: "-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Djava.rmi.server.hostname=127.0.0.1 -Dcom.sun.management.jmxremote.rmi.port=1099"
-      JMX_PORT: 1099
-
-Jconsole can now connect at ```jconsole 192.168.99.100:1099```
+```
+KAFKA_ADVERTISED_LISTENERS=SSL://_{HOSTNAME_COMMAND}:9093,PLAINTEXT://9092
+```
 
 ## Listener Configuration
```

Comment:
Moved this listener information next to the advertised host for locality of reference.